qCC/db_tree/ccDBRoot.cpp

```diff
@@ -1700,6 +1704,7 @@ void ccDBRoot::selectByTypeAndName()
 	//For generic-only types the match type gets overridden and forced to
 	//false because exclusive match makes no sense!
 	bool exclusive;
+    bool regex;
```

Comment:
I would recommend that, as a general practice, _all_ variables are initialized at declaration (when possible).

Then you know it's done, it signals to other developers what the intention is, and it save other devs from having to parse a bunch of code in their head.

It will save a lot of time and bug hunting - I guarantee it!